src/types/onyx/OnyxUpdatesFromServer.ts

```diff
@@ -2,9 +2,11 @@ import {OnyxUpdate} from 'react-native-onyx';
 import Request from './Request';
 import Response from './Response';
 
+type OnyxServerUpdate = OnyxUpdate & {shouldNotify?: boolean};
```

Comment:
Not sure about this update, maybe it makes sense to update `OnyxUpdate` type in the onyx lib if we really need `shouldNotify` field

Reply:
@VickyStash If I remember correctly it was me that suggested to create this separate type. The reason for that is because the `OnyxUpdate` from Onyx itself doesn't have `shouldNotify`, but Expensify backend often sends Onyx updates with this property together, so that's why I advised @fvlvte  to create a separate type.